src/lib/manager/aws/elasticache-manager.ts

```diff
@@ -0,0 +1,98 @@
+import * as elasticache from 'aws-cdk-lib/aws-elasticache'
+import * as common from '../../common'
+import * as types from '../../types'
+import * as utils from '../../utils'
+
+/**
+ * @stability stable
+ * @category cdk-utils.elasticache-manager
+ * @subcategory Compute
+ * @summary Provides operations on AWS Elasticache Service.
+ * - A new instance of scope class is injected into {@link common.CommonConstruct} constructor.
+ * - If a custom construct extends {@link common.CommonConstruct}, an instance is available within the context.
+ * @example
+ * import * as common from '@gradientedge/cdk-utils'
+ *
+ * class CustomConstruct extends common.common.CommonConstruct {
+ *   constructor(parent: cdk.Construct, id: string, props: common.CommonStackProps) {
+ *     super(parent, id, props)
+ *     this.props = props
+ *     this.elasticacheManager.createElasticache('MyElasticache', scope, props)
+ *   }
+ * }
+ *
+ * @see [CDK Elasticache Module]{@link https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-elasticache.CfnCacheCluster.html}
```

Comment:
(minor) Link the docs to V2 overview although it's in preview
https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_elasticache-readme.html